drivers/SmartThings/zigbee-smoke-detector/src/aqara/init.lua

```diff
@@ -0,0 +1,157 @@
+-- Copyright 2024 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+local data_types = require "st.zigbee.data_types"
+local clusters = require "st.zigbee.zcl.clusters"
+local cluster_base = require "st.zigbee.cluster_base"
+local battery_defaults = require "st.zigbee.defaults.battery_defaults"
+local capabilities = require "st.capabilities"
+
+
+local selfCheck = capabilities["stse.selfCheck"]
+local setSelfCheckStateCommandName = "setSelfCheckState"
+
+local PRIVATE_CLUSTER_ID = 0xFCC0
+local PRIVATE_ATTRIBUTE_ID = 0x0009
+local MFG_CODE = 0x115F
+local PRIVATE_MUTE_ATTRIBUTE_ID = 0x0126
+local PRIVATE_SELF_CHECK_ATTRIBUTE_ID = 0x0127
+local PRIVATE_SMOKE_ZONE_STATUS_ATTRIBUTE_ID = 0x013A
+local PowerConfiguration = clusters.PowerConfiguration
+
+
+local FINGERPRINTS = {
+    { mfr = "LUMI", model = "lumi.sensor_smoke.acn03" }
+}
+
+
+local CONFIGURATIONS = {
+  {
+    cluster = PRIVATE_CLUSTER_ID,
+    attribute = PRIVATE_SMOKE_ZONE_STATUS_ATTRIBUTE_ID,
+    minimum_interval = 1,
+    maximum_interval = 3600,
+    data_type = data_types.Uint16,
+    reportable_change = 1
+  },
+  {
+    cluster = PowerConfiguration.ID,
+    attribute = PowerConfiguration.attributes.BatteryVoltage.ID,
+    minimum_interval = 30,
+    maximum_interval = 3600,
```

Comment:
Are you sure you want a 1-hour max interval? We use a 6 hour default for battery voltage.

Since the reportable change is 1, the device _should_ change whenever the value does, these extra reports are most likely just draining your device's battery without much benefit.

Reply:
@Hongming6

Reply:
Yes,Sure.The battery life of the product can reach 2-3 years without any impact.